spec/fake_braintree/transaction_spec.rb

```diff
@@ -77,6 +87,13 @@
       result = Braintree::Transaction.refund(create_id('foobar'), '1')
       expect(result).to be_success
     end
+
+    it "sets the creation time" do
+      transaction = Braintree::Transaction.refund(create_id('foobar'), '1').transaction
+
+      creation_time = Time.parse(transaction.created_at)
+      expect(creation_time).to be_within(1).of(Time.now)
```

Comment:
Do not use `Time.now` without zone. Use one of `Time.zone.now`, `Time.now.current`, `Time.now.in_time_zone`, `Time.now.utc`, `Time.now.getlocal`, `Time.now.iso8601`, `Time.now.jisx0301`, `Time.now.rfc3339`, `Time.now.to_i`, `Time.now.to_f` instead.